### ISSUE TITLE
Provide contextual help for the rights_statement

### ISSUE THREAD

**Issue by scherztc** [uclibs/ucrate#319]
### Descriptive summary

We need some contextual help for the Rights field. 
This should be set in the config/locales files for each work type.
We will have to reference a product owner for the exact wording.

### Expected behavior

That we have some contextual help.

### Actual behavior

<img width="647" alt="screen shot 2018-07-26 at 11 41 04 am" src="https://user-images.githubusercontent.com/5084562/43273172-c23d2668-90c9-11e8-83b2-df3c61cb3d27.png">

### Steps to reproduce the behavior

1. Do this
1. Then do this...

### Related work

Link to related issues or prior related work here.

**Comment by crowesn** [comment]
I'm noticing that on develop we have both rights statement field and license field. Rights statement is redundant, no? Was supposed to be overriden by license maybe? 

<img width="688" alt="screen shot 2018-07-31 at 3 41 06 pm" src="https://user-images.githubusercontent.com/1069588/43482999-500d7634-94d8-11e8-8b86-593acd7ea635.png">

**Comment by hortongn** [comment]
@crowesn Yes, we do.  I'll look into it.  Might need to run it by the POs.  Hyrax has this field, but they don't provide contextual help text for it.

I'm marking this blocked until we develop text.

**Comment by hortongn** [comment]
We don't have a rights statement anymore

**Comment by Janell-Huyck** [comment]
Issues now handled by Jira.
